frontend/app/src/components/home/createOrUpdateGroup/CreateOrUpdateGroup.svelte

```diff
@@ -424,7 +426,7 @@
                     <div class="members">
                         <ChooseMembers
                             userLookup={searchUsers}
-                            bind:members={candidateGroup.members}
```

Comment:
This was causing the candidate group to have multiple members, even though these users are only invited, they are not already members. So I've introduced a new variable to hold the invited users to avoid incorrectly modifying the candidate group members.